src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -254,13 +240,10 @@ const ChannelImage = ({
     return (
       <View>
         {renderMainImage()}
-        <View style={styles.postNotificationImage}>
-          <AnonymousIcon
-            fill={COLORS.holyTosca}
-            width={dimen.normalizeDimen(21)}
-            height={dimen.normalizeDimen(21)}
-          />
-        </View>
+        <FastImage
+          source={AnonymousProfile}
+          style={[styles.postNotificationImage, styles.anonPmNotificationImageContainer]}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `AnonymousIcon` component has been replaced with the `FastImage` component. Ensure that the `FastImage` component is correctly rendering the `AnonymousProfile` image and that it fits well within the design of the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->